scripts/plot_performance_breakdown.py

```diff
@@ -25,6 +25,7 @@
 
     sum_parse_duration = 0.0
     sum_sql_translation_duration = 0.0
+    sum_cache_duration = 0.0
```

Comment:
Could you please run benchmark_all so that we also get TPC-H MT and TPC-DS numbers?